google/cloud/bigquerycontrol/CMakeLists.txt

```diff
@@ -197,6 +197,10 @@ if (BUILD_TESTING AND GOOGLE_CLOUD_CPP_ENABLE_CXX_EXCEPTIONS)
                          PROPERTIES LABELS "integration-test;quickstart")
 endif ()
 
+if (BUILD_TESTING)
```

Comment:
Alternative: unconditionally add the subdirectory, and return from the subdirectory if `NOT BUILD_TESTING`.